src/main/kotlin/deployment/DeploymentRunner.kt

```diff
@@ -39,21 +38,29 @@ internal class DeploymentRunner(
         null
     }
     private val cohortPollingInterval: Long = getCohortPollingInterval()
+    // Fallback in this order: proxy, stream, poll.
+    private val amplitudeFlagConfigPoller = FlagConfigFallbackRetryWrapper(
+        FlagConfigPoller(flagConfigApi, flagConfigStorage, cohortLoader, cohortStorage, config, metrics),
+        null, config.flagConfigPollerIntervalMillis, 1000
+    )
```

Comment:
NIT: The standard newline formatting for kotlin is different from java (more like javascript) where each argument gets its own newline. E.g.

```
private val amplitudeFlagConfigPoller = FlagConfigFallbackRetryWrapper(
        FlagConfigPoller(flagConfigApi, flagConfigStorage, cohortLoader, cohortStorage, config, metrics),
        null, 
        config.flagConfigPollerIntervalMillis, 
        1000
    )
```